e2e/testdata/fn-render/out-of-place-fnchain-unwrap/.expected/exec.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+set -eo pipefail
+
+kpt fn render -o stdout \
```

Comment:
e2e test for unwrap output of render? This is just testing the unwrap for eval which is tested seperately.

Reply:
Ditto for testing directory output of render.

Reply:
Added all possible cases.